packages/react-components/src/date-input/src/DateRangeInput.tsx

```diff
@@ -109,12 +119,12 @@ const DateInput = forwardRef(({
 
     const dateProps = useDateInput({
         value,
-        placeholder,
+        //placeholder, // TODO: TS useDateInput doesnt use this property
```

Comment:
useDateInput doesn't use placeholder and autofocus, can i remove them from here?

Reply:
done